packages/core/src/utils.js

```diff
@@ -1,13 +1,16 @@
 import React from "react";
 import * as ReactIs from "react-is";
-import mergeAllOf from "json-schema-merge-allof";
 import fill from "core-js-pure/features/array/fill";
 import union from "lodash/union";
 import jsonpointer from "jsonpointer";
 import fields from "./components/fields";
 import widgets from "./components/widgets";
 import validateFormData, { isValid } from "./validate";
+import _ from "lodash";
```

Comment:
To keep the build size down, it's preferable to import just the necessary function similarly to line 4.

```suggestion
import memoize from "lodash/memoize";
```